content/uc-doc/contributors/integration-tests.md

```diff
@@ -0,0 +1,151 @@
+This document describes the conventional setup for per-component integration tests in wazo projects.
+
+## Requirements
+
+The integration tests rely directly on docker(or an equivalent drop-in) and docker-compose, and
+assume that functional [`docker-compose`](https://docs.docker.com/compose/) and
+[`docker`](https://docs.docker.com/) commands are available in the environment.
+
+Additional python dependencies(including test runners, frameworks and any tool and library used by
+the tests) are made available in a pip requirements.txt file, to be be installed into a specific
+virtual environment.
+
+## Structure
+
+(As a general guide on conventional project structure, see also
+[Wazo daemon file structure](https://github.com/wazo-platform/wazo-notebook/blob/master/file-structure.md)
+
+All files related to integration tests should be present under an `integration_tests` directory
+directly below a project root directory. `integration_tests/assets` should contain docker-compose
+yaml files as well as configuration and static assets required for tests at runtime(usually mounted
+as volumes configured in the compose files). `integration_tests/docker` should contain custom
+dockerfiles and other files required to build images for the test setup(referenced in compose
+files). `integration_tests/suite` should contain the actual python tests definitions, including any
+custom helpers and utilities. `integration_tests/test-requirements.txt` (or a differently named pip
+requirements file) should contain all python requirements required for the test environment, such as
+the test runner, and other tools and libraries used by the tests. `integration_tests/Makefile`
+should exist and define an interface to interact with the test setup using the `make` utility.
```

Comment:
Maybe you could format this paragraph by file (or tree structure) to facilitate the reading?

Reply:
Oups, was supposed to be one line per file.

Reply:
Oh yeah you're right. The linter just wrap everything together because in markdown, only one carriage return mean nothing